gradio/interface.py

```diff
@@ -178,27 +179,21 @@ def __init__(
         if not isinstance(outputs, list):
             outputs = [outputs]
 
+        if "state" in inputs or "state" in outputs:
```

Comment:
Will this work if someone passes in a `gr.Variable()` instead of `"state`"

Reply:
If they pass the same variable, yes. Meaning
```
var = gr.Variable()
gr.Interface(fn, ["text", var], ["text", var])
```
will work, but
```
gr.Interface(fn, ["text", gr.Variable()], ["text", gr.Variable()])
```
will not map to the same variable. Using the string "state" automatically creates the same Variable for both input and output.